oeps/oep-0002.rst

```diff
@@ -64,9 +64,15 @@ following fields:
     ``oep-n``). Each value would be one of ``True``, ``False``, or a
     reason dictionary. ``True`` and ``False`` would indicate accordance (or
     lack thereof) with the specified OEP. A reason dictionary
-    must have ``state`` and ``reason`` keys. The ``state`` key is a boolean,
-    and ``reason`` is a string that should explain why it's ok that the
-    ``state`` is ``False`` (this will be displayed in reporting tools).
+    must have a ``state`` key. The ``state`` key can have a value of ``True``,
+    ``False`` or ``Not Applicable``. If the ``state`` is ``False`` the reason
```

Comment:
It might be worth clarifying whether these are strings or booleans, and providing some sample snippets of yaml that are correctly formatted.